Classes/Service/WarnCellParser.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the package jweiland/weather2.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace JWeiland\Weather2\Service;
+
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+class WarnCellParser implements WarnCellParserInterface
+{
+    public function parse(string $data): array
+    {
+        $rawRows = GeneralUtility::trimExplode(PHP_EOL, $data, true);
+        array_shift($rawRows); // Remove header row
+
+        $rows = [];
+        foreach ($rawRows as $index => $rawRow) {
+            $fields = str_getcsv($rawRow, ';');
+            if (count($fields) !== 5) {
+                continue;
+            }
+
+            [$warnCellId, $name, $shortName, $sign] = $fields;
```

Comment:
I remember that "nuts" was on third position. Removing it will set KURZNAME from CSV to $shortName which looks good for me, and CCC is now $sign, which was BL before. Don't know, if that was a good move.